openslides/agenda/static/js/agenda/base.js

```diff
@@ -0,0 +1,122 @@
+(function () {
```

Comment:
Why do you added this closure? We work with angular modules which do the same thing. I don't think that we need double closures.